lib/samples/display_clusters/display_clusters_sample.dart

```diff
@@ -58,18 +61,33 @@ class _DisplayClustersSampleState extends State<DisplayClustersSample> {
   }
 
   void onMapViewReady() async {
+    // Get the power plants web map from the default portal.
+    final portal = Portal.arcGISOnline();
+    final portalItem = PortalItem.withPortalAndItemId(
+        portal: portal, itemId: '8916d50c44c746c1aafae001552bad23');
+    // load the portal item.
+    await portalItem.load();
+    // create a map from the portal item
+    _map = ArcGISMap.withItem(portalItem);
+    // set the map to the map view controller
     _mapViewController.arcGISMap = _map;
+    // load the map
     await _map.load();
+    // check if the map is loaded
     if (_map.loadStatus == LoadStatus.loaded && mounted) {
```

Comment:
I don't think we should check `loadStatus` and `mounted` here. We don't do this in any other sample.

Reply:
Shouldn't we still check if it is mounted or not? I though we use it instead of try/catch for our async fucntions (like load() here) or did I misunderstand @PaulAllanSturm